server/views/pages/licence/AP.njk

```diff
@@ -1,39 +1,77 @@
 {% extends "pages/licence/pdf-layout.njk" %}
 
-{% from "pages/licence/partials/title.njk" import title %}
-{% from "pages/licence/partials/offender.njk" import offender %}
-{% from "pages/licence/partials/objectives.njk" import objectives %}
-{% from "pages/licence/partials/supervision.njk" import supervision %}
-{% from "pages/licence/partials/induction.njk" import induction %}
-{% from "pages/licence/partials/releasedToOther.njk" import releasedToOther %}
-{% from "pages/licence/partials/conditions.njk" import conditions %}
-{% from "pages/licence/partials/cancellation.njk" import cancellation %}
-{% from "pages/licence/partials/recall.njk" import recall %}
-{% from "pages/licence/partials/failureToComply.njk" import failureToComply %}
-{% from "pages/licence/partials/signatures.njk" import signatures %}
-{% from "pages/licence/partials/footer.njk" import footer %}
-
 {# Data supplied to this template
    - imageData(offender image).
    - htmlPrint(true/false).
    - licence - already in context as is in res.locals (from fetchLicenceMiddleware).
-   - additionalConditons - string[] containing additional conditions with placeholders expanded with values.
+   - additionalLicenceConditons - string[] containing additional licence conditions with placeholders expanded with values.
 #}
 
 {% set offenderName = licence.forename + " " + licence.surname %}
 {% set pageTitle = offenderName %}
 
 {% block content %}
-    {{ title(offenderName) }}
-    {{ offender(licence.dateOfBirth, licence.licenceExpiryDate, licence.nomsId, imageData, htmlPrint) }}
-    {{ objectives() }}
-    {{ supervision(licence.licenceStartDate, licence.licenceExpiryDate) }}
-    {{ induction(licence.appointmentPerson, licence.appointmentAddress, licence.appointmentTime, licence.comTelephone) }}
-    {{ releasedToOther() }}
-    {{ conditions(licence.standardLicenceConditions, additionalConditions, licence.bespokeConditions) }}
-    {{ cancellation() }}
-    {{ recall() }}
-    {{ failureToComply() }}
-    {{ signatures(licence.approvedByName, licence.approvedDate, offenderName) }}
-    {{ footer(licence.nomsId, licence.pnc, licence.cro, licence.bookingNo, licence.typeCode, licence.id, licence.version, licence.prisonCode) }}
+    <div id="title" class="title">Licence for {{ offenderName }}</div>
```

Comment:
I still have a slight preference for the macro method - creating a simpler top-level document with all sections clearly partitioned into their own files, but as discussed, I'm not precious about it. This change is also clear to follow / maintain and lets see how the various licence types develop, overlap or share sections?